src/components/FooterBar/FooterBar.tsx

```diff
@@ -80,20 +96,35 @@ const getData = (configFile?: ConfigFile) => {
       ],
     },
   ];
+};
+
+const bottomRender = ({ label, to }: FooterColumnItemProps): React.ReactElement => (
```

Comment:
This looks like a react component to me. Should we keep the naming convention for this to be `BottomRender` or `bottomRender`?

Reply:
ya i will move the component out into another file and rename it to be `BottomRender`, together with all the nav link components.